awx_collection/plugins/modules/tower_workflow_launch.py

```diff
@@ -70,97 +95,118 @@
 
 - name: Launch a Workflow with parameters without waiting
   tower_workflow_launch:
-    workflow_template: "Test workflow"
+    name: "Test workflow"
     extra_vars: "---\nmy: var"
     wait: False
   delegate_to: localhost
   run_once: true
   register: workflow_task_info
```

Comment:
let's get rid of the `delegate_to`, `run_once`, and `register`. That it assuming a lot of other things about how the playbook is written and isn't always necessary.

You could do the `register` if you intend to show how to use the return data, but ideally the supported return data, and not the one-off deprecated return data.